lighthouse-cli/test/cli/cli-flags-test.js

```diff
@@ -28,4 +28,14 @@ describe('CLI bin', function() {
       assert.ok(optionsWithDescriptions.includes(opt), `cli option '${opt}' has no description`);
     });
   });
+
+  it('array values support csv', () => {
+    const flags = getFlags(
```

Comment:
once we figure out other feedback would love the `--window-size` and `blockedUrlPattern` counter examples here too :)

Reply:
`blockedUrlPattern` just b/c comma could be in a pattern? what if we could escape it?

Reply:
nvm lets not do that :)